project3/index1.html

```diff
@@ -25,254 +25,340 @@ <h2>With great power there must also come-great responsibility</h2>
 </header>
 
 <body>
```

Comment:
The `header` element on line 13 should be inside your `body` tag on line 27. The _only_ elements that can be direct children of your `html` element are `head` and `body` -- everything else in your file must exist inside one of those two tags.